cogs/auth.py

```diff
@@ -31,7 +32,7 @@
 _USER_SCHEMA = ['username', 'first', 'last', 'auth', 'token', 'email']
 _GROUP_SCHEMA = ['name']
 
-DEFAULT_AUTHMOD = 'moodle'
```

Comment:
We should probably extract this to being a config setting vs just hard coding it here. Although, as long as changing the default hard coding doesn't break the tests, it's not a huge issue.